### PR TITLE
Add x-checker-data and switch qtmqtt to tarball

### PR DESCRIPTION
**Pull request by flyingcakes85**

### CODE REVIEW

File: org.kde.labplot2.json [hunk 1]
```diff
@@ -79,10 +97,15 @@
             ],
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://github.com/qt/qtmqtt.git",
-                    "commit": "cade213f562049400da81e60762400df68d7c56b",
-                    "tag": "v5.15.2"
+                    "type": "archive",
+                    "url": "https://github.com/qt/qtmqtt/archive/refs/tags/v6.2.3.tar.gz",
```
Comment (gerlachs) [inline org.kde.labplot2.json:101]: the version is related to the Qt version. 6.2.3 is not working. We need 5.15.2.

Comment (flyingcakes85) [reply org.kde.labplot2.json:101]: Since the application needs specific version of Qt, is it fine if I just leave out Qt from auto updating?

Comment (gerlachs) [reply org.kde.labplot2.json:101]: sure.

Comment (flyingcakes85) [reply org.kde.labplot2.json]: Updated

**Pull request merged**